486
Traceback (most recent call last):
  File "train.py", line 81, in <module>
    actor_loss, critic_loss =  ppo.update(replay_buffer)
  File "/home/ssbl/Desktop/PPO/ppo.py", line 39, in update
    state_array, action_array, reward_array, logp_old_array, true_state_value_array, advantage_array = replay_buffer.sample(BATCH_SIZE)
  File "/home/ssbl/Desktop/PPO/replay_buffer.py", line 117, in sample
    sample_index = random.sample(total_legal_length, index)
  File "/home/ssbl/anaconda3/envs/ml/lib/python3.7/random.py", line 317, in sample
    raise TypeError("Population must be a sequence or set.  For dicts, use list(d).")
TypeError: Population must be a sequence or set.  For dicts, use list(d).
